packages/api-explorer/webpack.dev.config.js

```diff
@@ -2,7 +2,7 @@
 
  MIT License
 
- Copyright (c) 2020 Looker Data Sciences, Inc.
+ Copyright (c) 2021 Looker Data Sciences, Inc.
```

Comment:
Must not be lint checked

Reply:
I guess so. Need to look at that (in another PR)

Reply:
Actually, they are lint checked. I changed to 2021

Reply:
Just confirmed. `yarn lint-changed-fix` fixed it when I set it back to 2020.